docker/graphite-web/Dockerfile

```diff
@@ -12,6 +12,7 @@ WORKDIR /usr/local/src/graphite-web
 
 
 RUN sed -i -e 's/whitenoise/whitenoise==3.3.1/g' requirements.txt  && \
+  sed -i -e 's/cairocffi/cairocffi==0.9.0/g' requirements.txt  && \
```

Comment:
This is already in master

Reply:
oh, it wasn't when I sent the PR :sweat_smile: